experimental/dds/tree/src/EditLog.ts

```diff
@@ -42,12 +53,17 @@ export interface OrderedEditSet<TChange = unknown> {
 	 */
 	getIdAtIndex(index: number): EditId;
 
-	/**
+    /**
 	 * @returns the index of the edit with the given editId within this `OrderedEditSet`, or `undefined` if no such edit exists.
 	 */
 	tryGetIndexOfId(editId: EditId): number | undefined;
 
-	/**
+    /**
+	 * @returns the id of the edit at the given index within this 'OrderedEditSet', or `undefined` if such edit is not available.
+	 */
+	tryGetIdAtIndex(index: number): EditId | undefined;
```

Comment:
There are quite a few API changes. It seems like there should be more tests added or changed.